darwin/torch/dataset.py

```diff
@@ -239,6 +432,31 @@ def _map_annotation(self, index: int):
 
         target = []
         for obj in annotation:
-            target.append({"category_id": self.classes.index(obj["name"]),
-                           "segmentation": [convert_polygon_to_sequence(obj["polygon"]["path"])]})
-        return target
+            sequence = convert_polygon_to_sequence(annotation["polygon"]["path"])
+            if len(sequence) / 2 < 3:
```

Comment:
are we dividing by 2 because x and y are counted separately?

Reply:
Because `sequence` contains the polygon in a `[x1, y1, x2, y2, ..., xn, yn]` format, so the number of points of the polygon is `len(sequence) / 2`

Reply:
I added a comment, so it's more clear.